server/upackages.pas

```diff
@@ -282,13 +276,21 @@ procedure LoadPackageOrProject(const FileName: string);
         Exit;
 
       if UpperCase(ExtractFileExt(FileName)) = '.LPK' then
-        Root := Root.FindNode('Package');
+      begin
+        CompilerOptions := Root.FindNode('Package').FindNode('CompilerOptions');
+        RequiredPackages := Root.FindNode('RequiredPkgs');
+      end
+      else
+      begin
+        CompilerOptions := Root.FindNode('CompilerOptions');
+        RequiredPackages := Root.FindNode('ProjectOptions').FindNode('RequiredPackages');
+      end;
 
-      if not Assigned(Root) then
+      if not Assigned(CompilerOptions) then
         Exit;
 
-      LoadPaths;
-      LoadDeps;
+      LoadPaths(CompilerOptions);
+      LoadDeps(RequiredPackages);
```

Comment:
Instead, add individual checks here.

Reply:
Done. I also tested the extension running on vscode with a .lpi without these nodes. The extension does not break.